### ISSUE TITLE
`docker-compose.yml`: pull access denied for `fargeo/aher6-2`

### ISSUE THREAD

**Issue by PaulJWright** [archesproject/arches-her#1346]
Hello,

I am trying to spin up the docker container w/ docker compose, and I am being denied pull access for `fargeo/aher6-2`. Is there a private container registry I need details for?

```
➜  arches-her git:(master) docker compose up
WARN[0000] /Users/paulwright/Documents/work/RICHeS/arches-her/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion
[+] Running 1/1
 ! aher Warning pull access denied for fargeo/aher6-2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied                                                 10.2s
[+] Building 0.0s (0/0)
```

**Comment by aj-he** [comment]
Hi @PaulJWright - please try using branch dev/1.0.x

UPDATED: The Readme in this branch should provide info on using the docker configuration

**Comment by PaulJWright** [comment]
Hi @aj-he, apologies for not getting back to you. This worked, thank you. 

What is the justification behind building from `dev/1.0.x` and not `main` or a later dev branch? We are considering Arches and Arches-her as a potential platform, and would like to build the latest version

EDIT: Apologies, just seen the update!

**Comment by aj-he** [comment]
@PaulJWright yeah, using a "dev/" branch as the current working branch is a bit unusual. It's a pattern being used by the core Arches team to manage LTS and latest versions. The core team set the latest dev branch as default, but arches-her hasn't done this yet. Probably should!

**Comment by PaulJWright** [comment]
> @PaulJWright yeah, using a "dev/" branch as the current working branch is a bit unusual. It's a pattern being used by the core Arches team to manage LTS and latest versions. The core team set the latest dev branch as default, but arches-her hasn't done this yet. Probably should!

This makes sense. I think I was confused by `dev/1.0.x` and `dev/7.0.x` `dev/7.1.x` etc, but I presume `dev/1.0.x` is an arches-her version, whereas 5/6/7.x.x are related to the core arches version?
